pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -441,6 +439,7 @@ stdenv.mkDerivation (finalAttrs: {
      *  External libraries
      */
     (enableFeature withAlsa "alsa")
+    (optionalString withAribcaption "--enable-libaribcaption")
```

Comment:
maybe `enableFeature withAribcaption "libaribcaption"` would be better here?

Reply:
Done.